pyro/contrib/gp/kernels/kernel.py

```diff
@@ -52,3 +67,267 @@ def _slice_input(self, X):
             return X.unsqueeze(1)
         else:
             raise ValueError("Input X must be either 1 or 2 dimensional.")
+
+    def __add__(self, other):
+        return self.add(other)
```

Comment:
I'm not a huge fan of operator overloading; in my experience it tends to significantly reduce readability.  I'd prefer keeping combination kernels explicit: users should always have to write `Sum(k1, k2)`.

Reply:
So I will uncomment them and remove the relevant part in docs. We can enable them again in the future if necessary.